include/bootchooser.h

```diff
@@ -21,6 +21,16 @@ GQuark r_bootchooser_error_quark(void);
 gboolean r_boot_is_supported_bootloader(const gchar *bootloader)
 G_GNUC_WARN_UNUSED_RESULT;
 
+/**
+ * Get current booted bootname slot.
+ *
+ * @param config the RaucConfig
+ *
+ * @return bootname, NULL if detection failed
+ */
+gchar *r_boot_get_bootname(RaucConfig *config);
```

Comment:
The trailing ";" will be removed in the follow-up commit anyway, but better not not introduce it at all ;)

Reply:
good catch